ApiDoctor.Validation/OData/Parameter.cs

```diff
@@ -44,11 +44,21 @@ public Parameter()
         [XmlAttribute("Type"), ContainsType]
         public string Type { get; set; }
 
+        private bool _isNullable;
```

Comment:
nit: i'm trying to evolve this project to have consistent coding styles (it doesn't right now), but i'm going to try to enforce consistency on new code. ultimately i should look at enabling style enforcement...

the applicable guidelines for this section:

- member variables are not prefixed with an underscore. instead they are accessed with `this.memberName`
- prefer autoproperties where possible instead of seprate field + accessor property
- fields should go at the top of the file: fields, constructors, properties, then methods. within each grouping, public goes first, then protected, then internal, then private.

Reply:
Got it, thank you. Will fix.